data/report_templates/project_summary.md

```diff
@@ -213,7 +213,7 @@ such as publications, presentations, posters, etc., using the following statemen
 
 > The authors acknowledge support from the National Genomics Infrastructure in {% if project.ngi_facility -%} {{ project.ngi_facility }} {%- endif %}
 > funded by Science for Life Laboratory, the Knut and Alice Wallenberg Foundation and the Swedish Research Council,
-> and SNIC/Uppsala Multidisciplinary Center for Advanced Computational Science for assistance with massively parallel sequencing
+> and NAISS/Uppsala Multidisciplinary Center for Advanced Computational Science for assistance with massively parallel sequencing
```

Comment:
This [page](https://www.naiss.se/policies/acknowledge/) says `Acknowledge both NAISS and SNIC if resources were used before and after 2023-01-01.`

Reply:
Pär will check!

Reply:
@parlundin This is the other PR